convene-web/Gemfile.lock

```diff
@@ -37,6 +37,8 @@ GEM
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
+    active_record_union (1.3.0)
```

Comment:
The README indicates that for folks using PostgreSQL, there is a more potent toolkit called [`ActiveRecordExtended`](https://github.com/georgekaraszi/ActiveRecordExtended).

This may give us more super-powers for the future; thoughts on using that instead of the one that just adds `Union`?

Reply:
Nice, didn't know about this tool.